app/src/main/java/com/android/streetworkapp/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -20,15 +40,39 @@ fun BottomNavigationMenu(
     tabList: List<TopLevelDestination>
 ) {
 
-  NavigationBar(modifier = Modifier.testTag("bottomNavigationMenu"), containerColor = Color.Gray) {
-    tabList.forEach { topLevelDestination ->
-      NavigationBarItem(
-          modifier = Modifier.testTag("bottomNavigationItem"),
-          icon = {
-            Icon(topLevelDestination.icon, contentDescription = topLevelDestination.textId)
-          },
-          selected = false,
-          onClick = { onTabSelect(topLevelDestination) })
-    }
-  }
+  NavigationBar(
+      modifier = Modifier.testTag(BottomNavigationMenuType.DEFAULT.getTopLevelTestTag()),
+      containerColor = Color.Gray) {
+        tabList.forEach { topLevelDestination ->
+          NavigationBarItem(
+              modifier = Modifier.testTag("bottomNavigationItem"),
+              icon = {
+                Icon(topLevelDestination.icon, contentDescription = topLevelDestination.textId)
+              },
+              selected = false,
+              onClick = { onTabSelect(topLevelDestination) })
+        }
+      }
+}
+
+/**
+ * Bottom bar for the event screen, displaying a button to join the event.
+ *
+ * @param participants The number of participants that have joined the event.
+ * @param maxParticipants The maximum number of participants allowed in the event.
+ */
+@Composable
+fun EventBottomBar(participants: Int, maxParticipants: Int) {
+  BottomAppBar(
+      containerColor = Color.Transparent,
+      modifier = Modifier.testTag(BottomNavigationMenuType.EVENT_OVERVIEW.getTopLevelTestTag())) {
+        Row(horizontalArrangement = Arrangement.Center, modifier = Modifier.fillMaxWidth()) {
+          Button(
+              onClick = {},
```

Comment:
I think it would be good for this button to display a toast or something so that the assistants know that it is not broken. This was talked about in the last meeting with the coaches.

Reply:
For the add_event screen, please note that it has been fixed in my last merged PR, there is no need to change it again (it should be ok once merged)